server/system_manager.py

```diff
@@ -92,38 +95,38 @@ def main():
                 if time_since_activation > _ONLINE_SETTLING_TIME_SEC:
                     if robot.autonomy_hold == True:
                         if time.time() - acorn_activated_time > _MAXIMUM_TIME_TO_ATTEMPT_AUTONOMY_SEC:
-                            print("Exceeded time to attempt autonomy but autonomy hold still active.")
+                            print(
+                                "Exceeded time to attempt autonomy but autonomy hold still active.")
                             time.sleep(_LONG_PAUSE_SEC)
                             continue
                         print("Clearing Autonomy Hold")
-                        redis_utils.clear_autonomy_hold(redis_client=r, vehicle_name=robot.name, value=True, active_site=active_site)
+                        redis_utils.clear_autonomy_hold(
+                            redis_client=r, vehicle_name=robot.name, value=True, active_site=active_site)
                         time.sleep(_LONG_PAUSE_SEC)
                         continue
                     if AUTONOMY_AT_STARTUP:
                         if robot.control_state == CONTROL_ONLINE:
                             print("Activating Autonomy")
-                            redis_utils.set_vehicle_autonomy(redis_client=r, vehicle_name=robot.name, speed=float(AUTONOMY_SPEED), enable=True, active_site=active_site)
+                            redis_utils.set_vehicle_autonomy(redis_client=r, vehicle_name=robot.name, speed=float(
+                                AUTONOMY_SPEED), enable=True, active_site=active_site)
```

Comment:
The formatter does its best, but such line breaks are not optimal - manual tune is required as drive-by of other changes.